python/tvm/relay/op/_transform.py

```diff
@@ -590,18 +593,21 @@ def squeeze_shape_func(attrs, inputs, _):
     """
     axis = attrs.axis if attrs.axis is None else get_const_tuple(attrs.axis)
     keep_axes = []
+    remove_axes = []
     if axis is not None:
         for i in range(inputs[0].shape[0].value):
             if i not in axis:
                 keep_axes.append(i)
+            else:
+                remove_axes.append(i)
```

Comment:
This could be written in a more concise form like `keep_axes.append(i) if i not in axis else remove_axes.append(i)`.

Reply:
Fixed

Reply:
This form can't pass lint because no assignment, I have to keep it.